docs/operations/create_constraint.mdx

```diff
@@ -17,6 +17,7 @@ Required fields: `name`, `table`, `type`, `up`, `down`.
     "columns": ["column1", "column2"],
     "type": "unique"| "check" | "foreign_key",
     "check": "SQL expression for CHECK constraint",
+    "no_inherit": "true|false",
```

Comment:
Should this also mention that it's only valid for `CHECK` constraints?

Reply:
We don't mention it in the description of `constraint`, so I omitted it here as well.